samples/Server/src/routes/addUserToRoom.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as express from 'express';
+import { getResourceConnectionString } from '../lib/envHelper';
+import { RoomsClient, RoomParticipant, Role } from '@azure/communication-rooms';
+
+const router = express.Router();
+interface AddUserToRoomParam {
+  userId: string;
+  roomId: string;
+  role: string;
+}
+
+/**
+ * route: /addUserToRoom
+ *
+ * purpose: Add the user to the room with given roomId and role.
+ *
+ * @param userId: id of the user as string
+ * @param roomId: roomId to add user as string
+ * @param role: role of user as string
+ */
+
+router.post('/', async function (req, res, next) {
+  // create RoomsClient
+  const roomsClient: RoomsClient = new RoomsClient(getResourceConnectionString());
+
+  const addUserToRoomParam: AddUserToRoomParam = req.body;
+
+  // request payload to add participants
+  const addParticipantsList: RoomParticipant[] = [
+    {
+      id: { communicationUserId: addUserToRoomParam.userId },
+      role: addUserToRoomParam.role as Role
+    }
+  ];
+
+  const addParticipants = await roomsClient.addParticipants(addUserToRoomParam.roomId, addParticipantsList);
```

Comment:
similar to my other comment on `createRoom`, `addParticipants` is what has happened, but not what this variable is

Reply:
Actually what does this return?

Reply:
It returns void. No need a variable and just sending a 201. Thanks.